findLargest.py

```diff
@@ -0,0 +1,6 @@
+defNo = 0
+print('Before :' , defNo)
+for num in [74,54,63,98,41,28,74,63,85,98]:
+    if num > defNo:
+        defNo = num
+print('Largest Number: ',defNo)

```

Comment:
Can you turn this into a "fix this broken code" exercise and put it to `loops.md`?